pillar/apps/odlvideo.sls

```diff
@@ -0,0 +1,154 @@
+# -*- mode: yaml -*-
+{% set app_name = 'odlvideo' %}
+{% set python_bin_dir = '/usr/local/pyenv/versions/3.6.3/bin' %}
+{% set ENVIRONMENT = salt.grains.get('environment', 'dev') %}
+{% set aws_creds = salt.vault.read('aws-mitx/creds/odlvideo-{env}'.format(env=ENVIRONMENT) %}
+{% set pg_creds = salt.vault.read('postgres-{env}-odlvideo/creds/odlvideo'.format(env=ENVIRONMENT) %}
+{% set youtube_creds = salt.vault.read('secret-odl-video/global/youtube-credentials') %}
+{% set app_cert = salt.vault.read('secret-odl-video/creds/mit-application-certificate') %}
+{% set cloudfront_key = salt.vault.read('secret-odl-video/global/cloudfront-private-key') %}
+
+{% set env_dict = {
+    'ci': {
+      'domain': 'video-ci.odl.mit.edu',
+      'log_level': 'DEBUG',
+      'use_shibboleth': False,
+      'ga_id': 'UA-108097284-1',
+      'transcode_pipeline_id': '1506027488410-93oya5',
+      'youtube_project_id': 'ovs-youtube-qa'
+      },
+    'rc-apps': {
+      'domain': 'video-rc.odl.mit.edu',
+      'log_level': 'INFO',
+      'use_shibboleth': True,
+      'ga_id': 'UA-5145472-27',
+      'transcode_pipeline_id': '1506081628031-bepkel',
+      'youtube_project_id': 'ovs-youtube-qa'
+      },
+    'production-apps': {
+      'domain': 'video.odl.mit.edu',
+      'log_level': 'WARN',
+      'use_shibboleth': True,
+      'ga_id': 'UA-5145472-27',
+      'transcode_pipeline_id': '1497541042228-8mpenl',
+      'youtube_project_id': 'ovs-youtube-production'
+      }
+} %}
+
+{% set env_data = env_dict[ENVIRONMENT] %}
+
+python:
+  versions:
+    - number: 3.6.3
+      default: True
+      user: root
+
+django:
+  pip_path: {{ python_bin_dir }}/pip3
+  django_admin_path: {{ python_bin_dir }}/django-admin
+  app_name: {{ app_name }}
+  settings_module: odl_video.settings
+  automatic_migrations: True
+  app_source:
+    type: git # Options are: git, hg, archive
+    revision: release
+    repository_url: https://github.com/mitodl/odl-video-service
+    state_params:
+      - branch: release
```

Comment:
How will `release-candidate` be handled?

Reply:
With care?...


I'll fix that.